splunk_add_on_ucc_framework/commands/modular_alert_builder/arf_template/default_html_theme/textarea.html

```diff
@@ -1,6 +0,0 @@
-{% extends "basecontrol.html" %}
-{% from "helper.html" import expand_ctrl_props with context %}
-{% block control %}
-<textarea name="action.{{ mod_alert.short_name }}.param.{{ param.name }}"
-		id="{{ mod_alert.short_name }}_{{ param.name }}" {{ expand_ctrl_props(param) }}></textarea>
-{% endblock %}
```

Comment:
where will textarea and other fields be supported in alerts?

Reply:
Made the changes. The HTML files for components are present in templates/html_files. The `textarea` wasn't documented and absent in `schema.json`, hence have updated the necessary details for the same.